strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/metadata/PypiArtifactMetadata.java

```diff
@@ -0,0 +1,242 @@
+package org.carlspring.strongbox.artifact.metadata;
+
+/**
+ * 
+ * @author ankit.tomar
+ *
+ */
+public class PypiArtifactMetadata
+{
+
+    private String comment;
+
+    private String metdataVersion;
+
+    private String fileType;
+
+    private String protcolVersion;
+
+    private String author;
+
+    private String homePage;
+
+    private String downloadUrl;
+
+    private String platform;
+
+    private String version;
+
+    private String description;
+
+    private String md5Digest;
+
+    private String action;
+
+    private String name;
+
+    private String license;
+
+    private String pyVersion;
+
+    private String summary;
+
+    private String authorEmail;
+
+    public String getComment()
+    {
+        return comment;
+    }
+
+    public PypiArtifactMetadata setComment(String comment)
```

Comment:
@anki2189 here and above setters should have `void` return type.  For "Builder Style" you better use `with` prefix instead of `set`. For example here it would be like this:
```
public void setComment(String comment)
{
    this.comment = comment;
}

public PypiArtifactMetadata withComment(String comment)
{
    this.comment = comment;
    return this;
}
```